File: Fundamentals of C programming/step06/6.3/6.3.03.c
/*

Написать программу проверяющую, является ли правильной дробь ABBA​.

Входные данные:
Два натуральных числа A,BA,B.

Выходные данные:
Строка yes, если дробь правильная, строка no в противном случае.

Sample Input:

84 47

Sample Output:

no

*/

#include <stdio.h>

int main() {
  int a,b;
  scanf("%d %d",&a,&b);
    
  if (a<b) {
      printf("yes\n");
  } else 
  printf("no\n");
    
  return 0;
}

File: Diving into JavaScript: For Beginners/step06/6.03/6.3.03.js
/*

Перед вами объект, который содержит значение основания. Добавьте в объект метод, который принимает от пользователя число Y, 
как параметр и возводит число Y - 2 в степень Y. Напишите инструкции ввода числа Y и вывода результата выполнения метода.

Sample Input 1:

4

Sample Output 1:

16

Sample Input 2:

5

Sample Output 2:

243

Sample Input 3:

3

Sample Output 3:

1

*/

let Y = Number(prompt());
const calculator = {
    base: Y - 2,
    power() {
        console.log(this.base**Y);
    }
};
calculator.power(Y);
